lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'menuFormatter.rb'
+require_relative 'order.rb'
+
+class Resturant
+
+  attr_reader :menu
+
+  def initialize
```

Comment:
Is this code easy to change? What if you wanted to create a new restaurant with a different menu?  How would you implement that?